modin/core/execution/dask/implementations/pandas_on_dask/partitioning/virtual_partition.py

```diff
@@ -396,6 +396,16 @@ def to_pandas(self):
         """
         return self.force_materialization().list_of_block_partitions[0].to_pandas()
 
+    def to_numpy(self):
```

Comment:
Can it be moved to the base class?

Reply:
I put this on the child classes because `to_pandas` is also defined here. It seems like it should be possible to move, since `force_materialization` and `list_of_block_partitions` are both defined on the `PandasDataframeAxisPartition` class, but I'd rather do that in a separate PR which moves `to_pandas` as well.